onnxruntime/core/graph/graph.cc

```diff
@@ -1964,8 +1965,17 @@ Status Graph::VerifyNodeAndOpMatch(const ResolveOptions& options) {
         node.op_ = nullptr;
       }
 
-      if (node.op_ && node.op_->HasFunction()) {
-        auto onnx_function_proto = node.op_->GetFunction();
+      if (node.op_ && (node.op_->HasFunction() || node.op_->HasContextDependentFunction())) {
+        const onnx::FunctionProto* onnx_function_proto;
+        onnx::FunctionProto temp;
```

Comment:
It seems to me that we don't need the temp here?
onnx::FunctionProto onnx_function_proto;

line 1973 
node.op_->BuildContextDependentFunction(function_body_ctx, onnx_function_proto);

Reply:
yea we don't it was for debugging when I was investigating crash dumps ... naive approach.